dart

```diff
@@ -0,0 +1,12 @@
+# this file would be "library/dart" in https://github.com/docker-library/official-images
```

Comment:
You probably want to ditch this file completely (maybe even just drop the commit introducing it), since you'll want to instead write a script to generate it (or just manually maintain it over in the official images repo, when that time comes).